### ISSUE TITLE
What next?

### ISSUE THREAD

**Issue by MasterOfTheTiger** [amazinigmech2418/UltraIDE#1]
What are the plans for this? I am not the best with JavaScript (but I am good at at). I am a lot better at websites in general though, and I love to make them. What can I help with?

**Comment by MasterOfTheTiger** [comment]
Oh and @amazinigmech2418

**Comment by AmazingMech2418** [comment]
Do you know python?

**Comment by MasterOfTheTiger** [comment]
Yes, but I am clueless when it comes to GUIs. I can make basic terminal programs.

**Comment by radioblahaj** [comment]
I'm also clueless at python, but can still mange

**Comment by MasterOfTheTiger** [comment]
I know Python, but not QT or GTK or whatever

**Comment by radioblahaj** [comment]
LOL. Not even me..

**Comment by MasterOfTheTiger** [comment]
Okay @amazinigmech2418, I am going to work on the online editor. I will build it in a different branch. I do need to know the theme in which we are going to be making it though.

**Comment by MasterOfTheTiger** [comment]
Like how do we want to make it look? Also, how are we supposed to do this? I don't think I know enough.  Maybe we can use some of the code from https://github.com/Nanomotion/xor/

**Comment by AmazingMech2418** [comment]
@MasterOfTheTiger It will just be a basic editor with all languages and conversions between languages.

**Comment by MasterOfTheTiger** [comment]
I don't know how on earth I would make syntax highlighting. @amazinigmech2418
